python/ray/rllib/env/entangled_env.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import gym
+from ray.rllib.utils.annotations import PublicAPI
+
+
+@PublicAPI
+class EntangledEnv(gym.Env):
+    """Interface for one physical environment that hosts
+    several logical environments."""
```

Comment:
How does this differ from VectorEnv?

Reply:
You have the point - it does not. My mistake, didn't realize there is env I could use out of the box.